etc/migration/12.1-to-12.2/3.dropEmailDomain.cql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "Tenant" drop "emailDomain" text;
```

Comment:
This fails because the column type `text` isn't part of the drop syntax. Also `drop` should be upper case.